lms/static/js/spec/main.js

```diff
@@ -181,22 +181,7 @@
             },
             'underscore': {
                 deps: ['underscore.string'],
-                exports: '_',
-                init: function(UnderscoreString) {
```

Comment:
@dan-f Right, that's why I said that "the whole mixin paradigm is something we should move away from". I don't know what you are suggesting here.